### DISCUSSION TITLE
Is there any design toolkits available for WPF apps?

### DISCUSSION THREAD

**Discussion by roshannzth** [dotnet/wpf#8915] [Q&A]
For a designer (without programming knowledge) to create designs for wpf apps, do we have anything on figma similar to Windows UI 3 design file? or does the designer have to use VS blend on windows?

What is the best way to create designs for wpf?

**Comment by lindexi** [comment]
I find [FigmaSharp.Views.Wpf](https://github.com/microsoft/FigmaSharp/tree/main/FigmaSharp.Views/FigmaSharp.Views.Wpf )

**Reply by roshannzth** [reply]
i did look into this, its not actively maintained. and there are few broken links, not sure i will get support there if i face any issues.

so we went back to the older way, which is designer will customize using existing web ui kits and i will style it to match the design the hard way. (basically following our existing product design language instead of windows design language)

**Comment by miloush** [comment]
To clarify, are you looking for a set of UI elements that you could use in Figma to compose a WPF UI, or are you looking for a way to generate WPF code from a design in Figma?

**Reply by roshannzth** [reply]
my first impression of wpf after working on win ui was,
there would be design templates with windows design language
and also plugins for figma which will get me the xaml code(This is optional).

The goal here was to have exact design be implemented( i guess blend comes into picture).
And our designer is restricted to Mac and Figma, So I was looking at available options.

Anyways we are going with our product design language which will involve lot of customization.

it would have helped if we were following windows design template.
